sqlglot/expressions.py

```diff
@@ -6563,6 +6563,29 @@ def or_(
     return t.cast(Condition, _combine(expressions, Or, dialect, copy=copy, **opts))
 
 
+def xor_(
```

Comment:
The underscore is used eg. in `or_` because `or` is a reserved keyword

```suggestion
def xor(
```